docs/components/Banner/Web.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Banner } from "@jobber/components/Banner";
+
+export default {
+  title: "Components/Status and Feedback/Banner/Web",
+  component: Banner,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+  },
+} as ComponentMeta<typeof Banner>;
+
+const BasicTemplate: ComponentStory<typeof Banner> = args => (
+  <Banner {...args}>Your splines are being reticulated</Banner>
+);
+export const Basic = BasicTemplate.bind({});
+Basic.args = {
+  type: "notice",
+};
+
+const ActionsTemplate: ComponentStory<typeof Banner> = () => (
+  <>
+    <Banner
+      type="notice"
+      primaryAction={{
+        label: "View Plans",
+        onClick: () => alert("Plans"),
+      }}
+      dismissible={false}
+    >
+      Your trial has been extended!
+    </Banner>
+    <Banner
+      type="error"
+      primaryAction={{
+        label: "Refresh",
+        onClick: () => alert("Refreshing"),
+      }}
+    >
+      Network is unavailable. Please check your internet connection.
+    </Banner>
+  </>
+);
+export const ActionsInBanners = ActionsTemplate.bind({});
```

Comment:
On Mobile it is `ActionInBanners` but here it is `ActionsInBanners` I think we should have them be the same name
<img width="210" alt="image" src="https://github.com/GetJobber/atlantis/assets/17303106/eef4549d-44db-4544-8ae6-c940cad4f4be">